tests/test_effects.py

```diff
@@ -92,32 +118,28 @@ def match_programs_to_robots(robots_program, state):
                 robot.program = robot_cards
 
 
-def read_commands_and_state():
-    commands = get_commands("tests/test_gear/commands.yaml")
-    state = get_start_state("tests/test_gear/map.json")
-    return commands, state
-
-
-def play_test_game():
+@pytest.mark.parametrize(
+        ("map", "commands"),
+        get_test_maps(),
+        )
+def test_play_game(map, commands):
     """
     Play the game with given map.
     """
-    commands, state = read_commands_and_state()
-    robots_program = get_test_cards(commands)
-    add_start_tile_number_to_robots(state)
-    match_programs_to_robots(robots_program, state)
 
-    apply_all_effects(state, registers=2)
+    read_commands = get_commands(commands)
+    state = get_start_state(map)
```

Comment:
Tady je zvláštní jak do funkce `get_commands` vstupuje proměnná `commands`, u které bych čekala podle názvu, že již povely obsahuje. Přitom obsahuje pouze cestu k souboru. A podobně i u mapy. Nebo to alespoň přidej do dokumentačního řetězce.

Reply:
Dalsi pohrobek, ano - provedu kontrolu nazvu, zda skutecne obsahuji, co maji.